src/components/Artists/ArtistsItem/ArtistItem.js

```diff
@@ -0,0 +1,31 @@
+import { MDCRipple } from "@material/ripple";
+import ArtistItemTemplate from "./ArtistItem.html";
+
+export class ArtistItemComponent {
+  constructor(mountPoint, artistData) {
```

Comment:
constructor structure is `mountPoint, props`

Reply:
Fixed